integration/conftest.py

```diff
@@ -310,12 +310,13 @@ def storage_nodes(reactor, temp_dir, introducer, introducer_furl, flog_gatherer,
     # start all 5 nodes in parallel
     for x in range(5):
         name = 'node{}'.format(x)
+        web_port=  9990 + x
```

Comment:
Ahh, can't do this.  Any pre-assigned port number is going to eventually collide with something and cause spurious test failures.

I don't know if any of the existing tools in Tahoe-LAFS can deal with this case.  There's a lot of complicated process management happeninghere.  Getting a pre-allocated *port* into the right process is going to be a challenge... but maybe not impossible.  It's probably easier to discuss this interactively if the path isn't clear (it's not 100% clear to me right now).

Reply:
This is definitely not the only hard-coded port in the integration tests..